bin/logstash

```diff
@@ -60,5 +60,6 @@ else
   for J in $(cd "${LOGSTASH_JARS}"; ls *.jar); do
     CLASSPATH=${CLASSPATH}${CLASSPATH:+:}${LOGSTASH_JARS}/${J}
   done
+  JAVA_OPTS=`exec "${JAVACMD}" -cp "${CLASSPATH}" org.logstash.launchers.JvmOptionsParser "$LOGSTASH_HOME"`
```

Comment:
moving the config file parsing to `bin/logstash` _removes_ it from all of the other command-line executables that rely on `logstash.lib.sh`. Is this desired?

Reply:
You are right, is not desired, I move the parsing of options file back in `logstash.lib.sh`